src/backend/marsha/core/utils/xmpp_utils.py

```diff
@@ -227,3 +227,24 @@ def add_jwt_token_to_url(url, token):
     generated_url[4] = urlencode(generated_query_string)
 
     return urlunparse(generated_url)
+
+
+def broadcast_message(room_name, message):
```

Comment:
IMO an event is also needed. The event must reflect what is made (starting a shared_live_media, ending it, changing its page)